src/foam/u2/view/ScrollTableView.js

```diff
@@ -85,6 +85,15 @@
       documentation: `
         A reference to the table element we use in the fitInScreen calculations.
       `
+    },
+    {
+      type: 'Int',
+      name: 'accumulator',
+      value: 0,
+      documentation: `Used internally to mimic native scrolling speed.`,
```

Comment:
Faster to use ' instead of `, plus no need since it is just a single line string.

Reply:
What makes it faster? I wouldn't expect there to be a performance difference.